index.js

```diff
@@ -34,9 +34,7 @@ module.exports = {
 				ecmaVersion: 2021,
 				sourceType: 'module',
 			},
-			plugins: [
-				'unicorn',
-			],
+			plugins: ['unicorn'],
```

Comment:
It is fixed by `yarn xo --fix`.

Reply:
I don't think `xo` do this, you must enabled prettier or something in your editor.